src/appleseed/foundation/math/scalar.h

```diff
@@ -178,6 +178,10 @@ T cube(const T x);
 template <typename T>
 T rcp(const T x);
 
+// Return 1 / x or eps if the absolute value of x is less than eps.
+template <typename T>
+T safe_rcp(const T x, const T eps);
```

Comment:
```
T safe_rcp(const T x, const T eps = default_eps<T>());
```

Reply:
I cannot do that because default_eps is not defined at this point.
I am also not sure if default_eps is the correct default value.